src/test/java/org/opensearch/jobscheduler/rest/RestGetJobTypeActionTests.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.jobscheduler.rest;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.common.bytes.BytesArray;
+import org.opensearch.common.xcontent.XContentType;
+import org.opensearch.jobscheduler.model.JobDetails;
+import org.opensearch.rest.RestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.test.OpenSearchTestCase;
+import org.opensearch.test.rest.FakeRestChannel;
+import org.opensearch.test.rest.FakeRestRequest;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.mockito.Mockito.mock;
+
+public class RestGetJobTypeActionTests extends OpenSearchTestCase {
+
+    private RestGetJobTypeAction action;
+
+    public HashMap<String, JobDetails> jobDetailsHashMap;
+
+    @Before
+    public void setUp() throws Exception {
+        super.setUp();
+        jobDetailsHashMap = new HashMap<>();
+        action = new RestGetJobTypeAction(jobDetailsHashMap);
+    }
+
+    @Test
+    public void getNameTests() {
+        String name = action.getName();
+        Assert.assertEquals("get_job_type_action", name);
+    }
+
+    @Test
+    public void getRoutes() {
```

Comment:
should have "test" in the name somewhere.

Reply:
ACK